cli/src/main/java/com/devonfw/tools/ide/commandlet/InstallCommandlet.java

```diff
@@ -29,7 +30,12 @@ public InstallCommandlet(IdeContext context) {
     super(context);
     addKeyword(getName());
     this.tool = add(new ToolProperty("", true, "tool"));
-    this.version = add(new VersionProperty("", false, "version"));
+    this.version = add(new VersionProperty("", false, "version",
+        (VersionIdentifier v, ValidationState state) -> {
```

Comment:
For lambdas it is convenient to use type inference:
```suggestion
        (v, state) -> {
```